mifosng-android/src/main/java/com/mifos/mifosxdroid/online/DashboardActivity.java

```diff
@@ -79,7 +79,7 @@ public void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_dashboard);
 
         ButterKnife.bind(this);
-        runJobs();
+        //runJobs();
```

Comment:
@robustTechie any reason why are we commenting this? 🙂

Reply:
actually it was not working at that time

Reply:
Okay, cool